From 2f896e1c518c7678fb059e16dd92bbac1cb68c95 Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>
Date: Tue, 26 Jun 2018 06:49:07 -0300
Subject: [PATCH] MAINTAINERS: fix location of ina2xx.txt device tree file
Git-commit: 2f896e1c518c7678fb059e16dd92bbac1cb68c95
Patch-mainline: v4.19-rc1
References: bsc#1051510

This file got moved and merged, causing the old reference to
not exist anymore. Fix it.

Fixes: 6e24d205a8aa ("hwmon: ina209: move binding docs to proper place")
Fixes: 62bc9f15e443 ("dt-bindings: merge ina209 binding into ina2xx binding")
Signed-off-by: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 0aaaff932c46..bc0d050d99fb 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7033,7 +7033,7 @@ M:	Guenter Roeck <linux@roeck-us.net>
 L:	linux-hwmon@vger.kernel.org
 S:	Maintained
 F:	Documentation/hwmon/ina209
-F:	Documentation/devicetree/bindings/i2c/ina209.txt
+F:	Documentation/devicetree/bindings/hwmon/ina2xx.txt
 F:	drivers/hwmon/ina209.c
 
 INA2XX HARDWARE MONITOR DRIVER
-- 
2.18.0

